inspirehep/config.py

```diff
@@ -85,6 +85,12 @@ def _(x):
 CACHE_TYPE = "redis"
 ACCOUNTS_SESSION_REDIS_URL = "redis://localhost:6379/2"
 
+# Mathoid
+# =======
+MATHOID_SERVER = os.environ.get(
```

Comment:
Shall we set this to `None` as default and have a check in the receiver for people who are not running mathoid locally?